src/config/dynamic-theme-fixes.config

```diff
@@ -8342,6 +8342,7 @@ dynadot.com
 
 INVERT
 .navbar-logo
+.signin-form-content-new img
```

Comment:
after some analysis I propose to remove both classes from INVERT section and add:
`img[data-src="/tr/mainsite2023/navbar-logo-dark-2023.png"]`

As I found all logos uses the same image.

And as a bonus I would add:
`.icons-line img`
which would inver some icons in footer

Reply:
Updated as suggested :)